### DISCUSSION TITLE
Applying Heom after Rotating Wave Approximation

### DISCUSSION THREAD

**Discussion by SanderStammbach** [qutip/qutip#2558] [Q&A]
Hey guys, 

I was trying to apply for Rotating Wave Approximation (RWA) at qutip.Heom. In my case, I want to look at qubit which is coupled to a single bosonic Bath. I wanted to use the BosonicBath package and deal with two different coupling operators.  My question is: Is there a way to implement HEOM with RWA similar to the work from the paper arXiv:2003.06134v3 and PyHEOM?

**Comment by nwlambert** [comment]
Sorry for the slow reply.  At the moment we don't explicitly support it, though it is supported in the Julia version 
https://qutip.org/HierarchicalEOM.jl/stable/bath_boson_RWA/bosonic_bath_RWA_intro/

Will add it to the todo list for the python version!  it also may be possible to bodge it with the current implementation, using two baths with appropriate correlation functions. But there may be some assumption in the way we formulate the heom in the python version that makes it hard or impossible without changing something a bit deeper. I will try and check and get back to you.
